### ISSUE TITLE
Package org.mapsforge.map.rendertheme is accessible from more than one module: mapsforge-map, mapsforge-themes

### ISSUE THREAD

**Issue by AKMANSoft** [mapsforge/mapsforge#1135]
I'm using mapforge verion 0.12.0 and my java version is JDK 11. Its because of same package name in mapsforge.map and mapsforge.themes

> The package org.mapsforge.map.rendertheme is accessible from more than one module: mapsforge.map, mapsforge.themes

````
TileRendererLayer tileRendererLayer = new TileRendererLayer(tileCache, mapDataStore, mapViewPosition, false,
				true, false, GRAPHIC_FACTORY, hillsRenderConfig) {
		@Override
		public boolean onTap(LatLong tapLatLong, Point layerXY, Point tapXY) {
			//My code block 
			return true;
		}
		};
tileRendererLayer.setXmlRenderTheme(InternalRenderTheme.DEFAULT);
````

**Comment by devemux86** [comment]
For questions, reports and discussion please always use the [forum](https://groups.google.com/group/mapsforge-dev).

**Comment by Asquator** [comment]
Any solution for the package split?

**Comment by devemux86** [comment]
Fixed by #1486
